pdf_on_submit/pdf_on_submit/doctype/pdf_on_submit_settings/pdf_on_submit_settings.json

```diff
@@ -7,6 +7,7 @@
  "field_order": [
   "section_break_1",
   "enabled_for",
+  "timeout_seconds",
   "section_break_8",
   "create_pdf_in_background"
```

Comment:
Found one more issue, sorry.

`timeout_seconds` is only relevant if `create_pdf_in_background` is enabled. Therefore, `timeout_seconds` should be shown in the same section, after `create_pdf_in_background`. And we can make it visible only if `create_pdf_in_background` is enabled (`"depends_on": "create_pdf_in_background"`).